app-connector-client/README.md

```diff
@@ -5,66 +5,32 @@
 # App Connector Client
 [![npm version](https://badge.fury.io/js/varkes-app-connector-client.svg)](https://badge.fury.io/js/varkes-app-connector-client)
 
-App Connector Client provides an interface to pair your application with Kyma and register it's APIs.
+App Connector Client provides an interface to pair your application with Kyma, register APIs and send Events.
 
 After startup, you have access to:
 - UI - http://localhost:10000
 - rest API - http://localhost:10000/metadata
 - console - http://localhost:10000/console
 
-## Starting the application
+## Run mock
 
-### Run local
-
-To run it local, specify a config file (as located in the /test folder) like that:
+To run it local, specify the [varkes config](https://github.com/kyma-incubator/varkes/blob/master/app-connector-client/test/varkes_config.json) file:

```

Comment:
Should be `run locally` - will correct this after your suggestions. Also `varkes config` (formatting)

Reply:
Please change it fully to:

## development
To run it local using an empty configuration
`npm start`

To run it local having debug logs enabled and an example configuration taken from the test suite:
`npm run start:dev`

Reply:
Done, had it in my notes :D and forgot to add.